Plugins/OutText.Internal.ps1

```diff
@@ -98,6 +98,12 @@
                 ## Fix Set-StrictMode
                 if (Test-Path -Path Variable:\Options) { $options = Get-Variable -Name Options -ValueOnly; }
                 else { $options = New-PScriboTextOptions; }
+
+                try{
+                    $options.SectionSeparator | Out-Null
+                }catch{
+                    $options | Add-Member -MemberType NoteProperty -Name SectionSeparator -Value '-'
+                }  
```

Comment:
OK. As the $Options variable is a hashtable we can check to see if there's a key set rather than using an expensive try/catch. In addition, we only need to check if there are existing options set (we know New-PScriboTextOptions will be OK). Can you change the `begin` script block to:
```powershell
            begin {
                ## Fix Set-StrictMode
                if (Test-Path -Path Variable:\Options) {
                    $options = Get-Variable -Name Options -ValueOnly;
                    if (-not ($options.ContainsKey('SectionSeparator'))) {
                        $options['SectionSeparator'] = '-';
                    }
                }
                else { $options = New-PScriboTextOptions; }
            }
```